### PR TITLE
Fix markdown in the Workspace controls sheet

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

This pull request (PR) fixes markdown in the Workspace controls sheet
image

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- no

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- no

## Affected documentation pages

_List HTML links for affected pages on <https://docs.magento.com/user-guide/>._

- https://docs.magento.com/user-guide/catalog/products.html

### CODE REVIEW

File: src/catalog/products.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ _Products Grid_
...
+|Add Product|Initiates the process to create a new simple product. To choose a specific product type, click the down arrow. Options: [Simple Product]({% link catalog/product-create-simple.md %}) / [Configurable Product]({% link catalog/product-create-configurable.md %}) / [Grouped Product]({% link catalog/product-create-grouped.md %}) / [Virtual Product]({% link catalog/product-create-virtual.md %}) / [Bundle Product]({% link catalog/product-create-bundle.md %}) / [Downloadable Product]({% link catalog/product-create-downloadable.md %}) / [Gift Card]({% link catalog/product-gift-card.md %})</span>|
```
Comment (BarnyShergold) [inline src/catalog/products.md:20]: I think the closing tag also needs removing.
```suggestion
|Add Product|Initiates the process to create a new simple product. To choose a specific product type, click the down arrow. Options: [Simple Product]({% link catalog/product-create-simple.md %}) / [Configurable Product]({% link catalog/product-create-configurable.md %}) / [Grouped Product]({% link catalog/product-create-grouped.md %}) / [Virtual Product]({% link catalog/product-create-virtual.md %}) / [Bundle Product]({% link catalog/product-create-bundle.md %}) / [Downloadable Product]({% link catalog/product-create-downloadable.md %}) / [Gift Card]({% link catalog/product-gift-card.md %})|
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]

**Comment by EvheniiaKolkhiieva** [comment]
Hi @BarnyShergold , thank you for reviewing. I have implemented your suggestion, could you please approve this PR?
Thank you!

**Review by BarnyShergold — APPROVED** [review]
Nice! BTW if you agree a change that's simple it's easier to PR again if you accept the suggestion on here rather than making the changes locally and re-pushing.

**Comment by EvheniiaKolkhiieva** [comment]
Oh @BarnyShergold  what a good observation! I never tried to do that, thank you, I will definitely try 😊

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the pull request successfully imported.

**Pull request merged**